app/index.js

```diff
@@ -22,25 +23,38 @@ process.chdir(__dirname);
 // Setup the app
 var port = process.env.PORT || 3000;
 app.listen(port);
-app.use(bodyParser.urlencoded({
-	extended: true
-}));
+app.use(
+  bodyParser.urlencoded({
+    extended: true
+  })
+);
 app.use(bodyParser.json());
 
 let spaces = 4;
-if(process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production') {
   spaces = 0;
 }
 app.set('json spaces', spaces);
 
 // Set the version
 app.set('version', require('../package.json').version);
 
+exec(`git rev-list -n1 ${app.get('version')}`, (err, stdout, stderr) => {
```

Comment:
This particular command will fail if the value in package.json isn't a tag in git, even though there still may be a commit we'd want to show. Instead, let's not provide a version at all and make the command:

```javascript
exec(`git rev-list -n1`, (err, stdout, stderr) => { ... });
```
which will be more robust